### PR TITLE
Update world protocol version

### PR DESCRIPTION
**Pull request by Gaprix**

### CODE REVIEW

**Comment by dries-c** [comment]
I'm not sure if this makes sense given the bedrock data hasn't changed at all

File: src/data/bedrock/block/BlockStateData.php [hunk 1]
```diff
@@ -42,8 +42,8 @@ final class BlockStateData{
 	public const CURRENT_VERSION =
 		(1 << 24) | //major
 		(21 << 16) | //minor
-		(0 << 8) | //patch
-		(3); //revision
+		(2 << 8) | //patch
+		(2); //revision
```
Comment (dktapps) [inline src/data/bedrock/block/BlockStateData.php]: This is only supposed to change if the block palette was updated (and a new blockstate upgrade schema added)

**Review by dktapps — CHANGES_REQUESTED** [review]

File: src/data/bedrock/block/BlockStateData.php (hunk 1 above)
Comment (Gaprix) [reply src/data/bedrock/block/BlockStateData.php]: But at the same time, PocketMine will not be able to open the vanilla world 1.21.2 without this change

Comment (Gaprix) [reply src/data/bedrock/block/BlockStateData.php]: Or do you mean a change specifically in this class?

Comment (dktapps) [reply src/data/bedrock/block/BlockStateData.php]: This specific change is invalid

**Review by dktapps — APPROVED** [review]

**Review by IvanCraft623 — APPROVED** [review]

**Pull request merged**